client/src/store/middleware/oauthMiddleware.js

```diff
@@ -17,7 +17,7 @@ export default function oauthMiddleware({ dispatch, getStateIgnored }) {
   return next => action => {
     switch (action.type) {
       case `${O.oauthResponse}`:
-        tryToSignIn(dispatch, event);
+        tryToSignIn(dispatch, window.event);
```

Comment:
I'd want to double check that  window.event is the intention here. I'd see if Alexa has insight on this.

Reply:
Looks like we are good with this one and it should do exactly that